### PR TITLE
Completed html-forms exercise-david Gonzalez

### PR DESCRIPTION
**Pull request by David-Gonzalez11**
form

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    Coding Form
+  </title>
+</head>
+<body>
+
+  <form action="" method="get" class="form example">
+    <h1>Coding Form!</h1>
+
+    <div>
+      <label for="Email">Email</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:14]: This `label` is not properly configured to connect with its form control element. Make sure that the `for` and `id` attribute values match.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      <div>
+        <input type="email" name="Email" id="email-address" />
+      </div>
+    </div>
+
+    <div>
+      <div>
+        <label for="Why coding">Why is coding cool?</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:22]: This `label` is not properly configured to connect with its form control element. Make sure that the `for` and `id` attribute values match. Also, no spaces in the value here, that means multiple values

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+          <option value="">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+    </div>
+    <div>
+
+      <label for="Favorite Language">Favorite Language(choose one) </label>
```
Comment (codingcodymiller) [inline html-forms/index.html:35]: This `label` is not properly configured to connect with its form control element. Make sure that the `for` and `id` attribute values match. Also, no spaces in the value here, that means multiple values

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+      <div>
+        <input type="radio" name="Favorite Language" value="HTML"/>HTML
+        <input type="radio" name="Favorite Language" value="CSS"/>CSS
+        <input type="radio" name="Favorite Language" value="JavaScript"/>JavaScript
+      </div>
+    </div>
+    <div>
+      <p> Would you like to sign up for more information?</p>
+      <label> <input type="checkbox" name="SignUp!" /></label>
+      <label for="Sign me up!" >Sign me up!</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:45]: This `label` is not properly configured to connect with its form control element. Make sure that the `for` and `id` attribute values match. Also, no spaces in the value here, that means multiple values

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+      <div>
+        <div>
+          <label for="comments">Comments</label>
+        </div>
+        <textarea></textarea>
+      </div>
+      <label for="submit"></label>
```
Comment (codingcodymiller) [inline html-forms/index.html:52]: If it's empty, there is no need for a `label` element

File: html-forms/index.html [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+        <input type="radio" name="Favorite Language" value="JavaScript"/>JavaScript
+      </div>
+    </div>
+    <div>
+      <p> Would you like to sign up for more information?</p>
+      <label> <input type="checkbox" name="SignUp!" /></label>
+      <label for="Sign me up!" >Sign me up!</label>
+      <div>
+        <div>
+          <label for="comments">Comments</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:48]: This `label` is not properly configured to connect with its form control element. Make sure that the `for` and `id` attribute values match.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-forms/index.html (hunk 6 above)
Comment (David-Gonzalez11) [reply html-forms/index.html]: https://user-images.githubusercontent.com/24907191/151510197-30f5c829-d6d7-4305-a7e7-9a39d7ff8871.mov

File: html-forms/index.html [hunk 7]
```diff
@@ -1,62 +1,59 @@
...
-    </div>
 
-    <div>
       <div>
-        <label for="Why coding">Why is coding cool?</label>
+        <div>
+          <label for="Why coding">Why is coding cool?</label>
+        </div>
+        <div>
+          <select id="Why coding">
```
Comment (codingcodymiller) [inline html-forms/index.html:22]: `id` values can not have spaces, you should use `kebab-case`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-forms/index.html [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+          <select id="why-coding">
+            <option value="">Select an option</option>
+            <option value="interesting">Because it's interesting</option>
+            <option value="fun">Because it's fun</option>
+            <option value="powerful">Because it's powerful</option>
+          </select>
+        </div>
+      </div>
+      <div>
+        <label for="Favorite Language">Favorite Language(choose one)</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:31]: Make sure you check if anything else needs to be changed with the same problem when you get feedback 👀

File: html-forms/index.html [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+        <div>
+          <input
+            type="radio"
+            name="Favorite Language"
+            value="HTML"
+            id="Favorite Language" />HTML
```
Comment (codingcodymiller) [inline html-forms/index.html:37]: Make sure you check if anything else needs to be changed with the same problem when you get feedback 👀

File: html-forms/index.html [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+          <input type="radio" name="Favorite Language" value="CSS" />CSS
+          <input
+            type="radio"
+            name="Favorite Language"
+            value="JavaScript" />JavaScript
+        </div>
+      </div>
+      <div>
+        <p>Would you like to sign up for more information?</p>
+        <input type="checkbox" name="SignUp!" id="Sign me up" />
```
Comment (codingcodymiller) [inline html-forms/index.html:47]: Make sure you check if anything else needs to be changed with the same problem when you get feedback 👀

File: html-forms/index.html [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+        <label for="Sign me up">Sign me up!</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:48]: Make sure you check if anything else needs to be changed with the same problem when you get feedback 👀

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-forms/index.html (hunk 11 above)
Comment (codingcodymiller) [reply html-forms/index.html:48]: This still needs to be addressed. Spaces are not allowed in `id` values

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**